lib/pc/index.js

```diff
@@ -29,6 +29,22 @@ module.exports = (data, staticData) => {
         }
       }
 
+      const dimensions = dimensionCodec['minecraft:dimension_type']?.value
+      if (dimensions) {
+        data.dimensionsById = {}
+        data.dimensionsByName = {}
+        for (const { name, id, element } of dimensions) {
+          const n = name.replace('minecraft:', '')
+          const d = {
+            name: n,
+            minY: element.min_y,
+            height: element.height
+          }
+          data.dimensionsById[id] = d
```

Comment:
What will be values of this not in 1.19 ?

Is it possible to have default values for other versions?

Reply:
There is no dimensions data for versions below 1.19 but static data for older versions could be added to Minecraft-data I think.

Reply:
For now let's either:
* Hardcore some values for all versions that don't have dimensions data
* Or document this is only supported for 1.19 for now